README.md

```diff
@@ -467,18 +471,6 @@ NNCF is also available via [conda](https://anaconda.org/conda-forge/nncf):
 conda install -c conda-forge nncf
 ```
 
-### System requirements
```

Comment:
Didn't we agree to refer to OV system requirements and https://github.com/openvinotoolkit/nncf/blob/develop/docs/Installation.md#corresponding-versions ?

Reply:
done

Reply:
Link to requirements is part of https://docs.openvino.ai/2024/get-started/install-openvino.html that exists in instalation,md

Reply:
It's misleading because there you talk about OV system requirements not NNCF. I also want to see it on the main page, not hidden in other docs. So I suggest to clearly state that "NNCF corresponds to OV system requirements, link, and the matrix of corresponding versions of NNCF, OV, PT, and TF can be found in Installation.md#corresponding-versions". You can add it to the Installation section if you prefer to remove this section.

Reply:
Updated, please check

Reply:
let's refer to ./docs/Installation.md not ./docs/Installation.md#corresponding-versions to cover the system requirements